events/templates/events/update-event.html

```diff
@@ -44,7 +44,7 @@ <h1>Cheer↑</h1>
             </div>
 
             <div class="form-group">
-                <input type="hidden" id="selected-location-id" name="event_location_id">
+                <input type="hidden" id="selected-location-id" name="event_location_id" value="{{ event.event_location.id }}">
                 <label for="autocomplete-input">Location:</label>
```

Comment:
I think this change causes an error when we create an event.
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/b6231968-82d1-4b99-90b2-2a4dc36f459f)